From 8dc162b2ae2e412c7d4e0d46f7a11001c34263dc Mon Sep 17 00:00:00 2001
From: Andrew Jones <drjones@redhat.com>
Date: Mon, 23 Aug 2021 18:57:11 +0200
Subject: [PATCH 1/4] hw/arm/virt: Remove 9.0 machine type
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Andrew Jones <drjones@redhat.com>
RH-MergeRequest: 38: hw/arm/virt: Remove 9.0 machine type
RH-Commit: [1/1 35c0734886622b88f6a715f13fba8f65331f7a82
RH-Bugzilla: 2002937
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Gavin Shan <gshan@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>

Testing: '/usr/libexec/qemu-kvm -M help' and sample VM run

Remove the 9.0 machine type for RHEL9 Beta to align with other
architectures. Also, like other architectures, the 8.5 machine
type should be the RHEL9 Beta default.

Signed-off-by: Andrew Jones <drjones@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/arm/virt.c | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/hw/arm/virt.c b/hw/arm/virt.c
index 3c8e6de36d..9e7cb687dc 100644
--- a/hw/arm/virt.c
+++ b/hw/arm/virt.c
@@ -3145,14 +3145,8 @@ static void rhel_machine_init(void)
 }
 type_init(rhel_machine_init);
 
-static void rhel900_virt_options(MachineClass *mc)
-{
-    compat_props_add(mc->compat_props, arm_rhel_compat, arm_rhel_compat_len);
-}
-DEFINE_RHEL_MACHINE_AS_LATEST(9, 0, 0)
-
 static void rhel850_virt_options(MachineClass *mc)
 {
-    rhel900_virt_options(mc);
+    compat_props_add(mc->compat_props, arm_rhel_compat, arm_rhel_compat_len);
 }
-DEFINE_RHEL_MACHINE(8, 5, 0)
+DEFINE_RHEL_MACHINE_AS_LATEST(8, 5, 0)
-- 
2.27.0

